cadc-vos/build.gradle

```diff
@@ -15,7 +15,7 @@ sourceCompatibility = 1.7
 
 group = 'org.opencadc'
 
-version = '1.1.8'
+version = '1.1.9'
```

Comment:
Looks like there were actually no changes in cadc-vos so you could leave the version as it.  I suspect there was a change originally when you had storageID directly in the Node class.

Reply:
You're right. Forgot to rollback the build number when I removed the Node changes. Will fix.

Reply:
Fixed.